plugins/dig/help.md

```diff
@@ -1,49 +1,40 @@
 # Description
 
-The DNS plugin is used for forward and reverse DNS lookups. This plugin uses [Dig](https://linux.die.net/man/1/dig), or Domain Information Groper, which is a network administration command-line tool for querying Domain Name System (DNS) name servers.
+The DNS plugin is used for forward and reverse DNS lookups. This plugin uses [Dig](https://linux.die.net/man/1/dig), or Domain Information Groper, which is a network administration command-line tool for querying Domain Name System (DNS) name servers
 
 # Key Features
-
-* Forward DNS lookup to find an IP address from a domain name
+  
+* Forward DNS lookup to find an IP address from a domain name  
 * Reverse DNS lookup to find a domain name from an IP address
 
 # Requirements
 
-_This plugin does not contain any requirements._
+# Supported Product Versions
+  
+* 2023-10-12
 
 # Documentation
 
 ## Setup
-
-_This plugin does not contain a connection._
+  
+*This plugin does not contain a connection.*
 
 ## Technical Details
 
 ### Actions
 
 #### Forward Lookup
-
-This action is used to request a forward lookup for a domain.
+  
+This action is used to request a forward lookup for a domain
 
 ##### Input
 
-It accepts a domain name of type `string` and one of the following record types:
-
-* A
-* AAAA
-* ANY
-* CNAME
-* MX
-* NS
-* PTR
-* SOA
-
```

Comment:
Is it okay to have this removed now? Not sure what it means & its very uncommon to see but lmk

Reply:
Is it the format that looks odd or the content? It's just the types of queries you can use in the required input

Reply:
yeah I would keep this, these represent the DNS query type for the dig command, eg

`dig example.com A` will return the ipv4 of exampe.com
`dig example.com AAAA` will return the ipv6 of exampe.com
`dig example.com MX` will return the mail exchange of exampe.com

and so on so these examples are worth keeping in

Reply:
As these the required fields that the user can enter are provided in the example below as part of the enum field this looks ok that it was removed

![image](https://github.com/rapid7/insightconnect-plugins/assets/144030336/3d184e1a-9568-4c44-ba5f-557115e189d8)